src/main/resources/anniv6Resources/localization/eng/GremlinTown/Potionstrings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "${ModID}:LouseMilk": {
+    "NAME": "Louse Milk",
+    "DESCRIPTIONS": [
+      "Draw #b{0} cards and gain #b{1} Energy."
+    ]
+  },
+  "${ModID}:PreerelxsBlueRibbon": {
+    "NAME": "Preerelx's Blue Ribbon",
+    "DESCRIPTIONS": [
+      "Gain #b{0} #yVigor"
+    ]
+  },
+  "${ModID}:RitualBlood": {
+    "NAME": "Ritual Blood",
+    "DESCRIPTIONS": [
+      "Impairs the Heart"
```

Comment:
```suggestion
      "#rImpairs the Heart."
```

Reply:
I have never seen red words in potion strings.  Is there precedent somewhere?

Reply:
not to my knowledge. Just think it adds some gravitas